config/locales/en.yml

```diff
@@ -27,5 +27,7 @@ en:
   lists:
     list:
       empty: This list is empty
+    show:
+      empty: This list is empty
```

Comment:
Even thought this is the same message as for the lists page, I would treat this as coincidental duplication. I think there are many reasons why the text in each place should differ in future.